### DISCUSSION TITLE
How to test abort in remap?

### DISCUSSION THREAD

**Discussion by kozliuk** [vectordotdev/vector#9574] [Q&A] [answered]
Hi guys, I have something like that in my transformer.
`if smth == "1" {
      some_var = "One"
    } else if smth == "2" {
      some_var = "Two"
    } else {
      abort
  }`

How to write unit test for else branch? I need to test abort case

**Comment by spencergilbert** [comment, marked as answer]
Hi @kozliuk you can use `no_outputs_from` - see an example here: https://github.com/vectordotdev/vector/blob/master/tests/behavior/transforms/route.toml#L25

**Reply by kozliuk** [reply]
Thanks. Will be cool add this to unit tests vector documentation
